### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by michaelkirk**
clarity edits/typos

### CODE REVIEW

**Review by charlesmchen — APPROVED** [review]
LGTM Can you please make similar changes in https://github.com/sqlcipher/sqlcipher/issues/255 

I'm trying to keep these docs in lockstep.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ### Summary
 
-iOS apps can't share an encrypted database with app extensions (e.g. share extensions) without being terminated every time they enter the background.
+iOS apps can't share an encrypted database with app extensions (e.g. share extensions) without being terminated every time they are suspsended.
```
Comment (charlesmchen) [inline README.md:5]: sp suspended.

**Comment by michaelkirk** [comment]
In d0bd9f0

File: README.md (hunk 1 above)
Comment (michaelkirk) [reply README.md:5]: done, thanks.

**Pull request closed without merging**